pandas/core/indexes/timedeltas.py

```diff
@@ -263,10 +263,11 @@ def union(self, other):
         -------
         y : Index or TimedeltaIndex
         """
-        self._assert_can_do_setop(other)
```

Comment:
same

Reply:
see related comment in DatetimeIndex module

Reply:
not sure what you are referring. you can simply remove this and t will hit the super method

Reply:
same comment. why is this neede?